paper/paper.md

```diff
@@ -52,29 +52,34 @@ bibliography: paper.bib
 
 # Summary
 
-Scientific software is usually created to analyze, model, and visualize data.
+Scientific software is usually created to acquire, analyze, model, and visualize data.
 As such, many software libraries include sample datasets in their distributions
 for use in documentation, tests, benchmarks, and workshops.
-The usual approach is to include smaller datasets in the GitHub repository
+A common approach is to include smaller datasets in the GitHub repository
 directly and package them with the source and binary distributions
 (e.g., scikit-learn [@scikit-learn] and scikit-image [@scikit-image] do this).
-Larger datasets require writing code to download the files from a remote server
+As data files increase in size, it becomes unfeasible to store them in GitHub
+repositories.
+Thus, larger datasets require writing code to download the files from a remote server
 to the user's computer.
 The same problem is faced by scientists using version control to manage their
 research projects.
-As data files increase in size, it becomes unfeasible to store them in GitHub
-repositories.
-While downloading a data file over HTTP can be done easily with modern Python
+While downloading a data file over HTTPS can be done easily with modern Python
 libraries, it is not trivial to manage a set of files, keep them updated, and
 check for corruption.
+For example, scikit-learn [@scikit-learn], Cartopy [@cartopy], and PyVista
+[@pyvista] all include code dedicated to this particular task.
 Instead of scientists and library authors recreating the same code, it would be
 best to have a minimalistic and easy to set up tool for fetching and maintaining
 data files.
 
 Pooch is a Python library that fills this gap.
-It manages a data *registry* by downloading files from one or more remote
-servers and storing them in a local data cache.
+It manages a data *registry* (containing file names, SHA256 hashes, and
```

Comment:
Hyphenated looks more common, but if so, make sure to be consistent in the whole document.

```suggestion
It manages a data *registry* (containing file names, SHA-256 hashes, and
```